src/org/exist/client/InteractiveClient.java

```diff
@@ -1309,18 +1346,15 @@ private synchronized boolean findRecursive(final Collection collection, final Pa
     /**
      * stores given Resource
      *
-     * @param fileName simple file or directory
+     * @param file file or directory
      * @throws XMLDBException
      */
-    protected synchronized boolean parse(String fileName) throws XMLDBException {
+    protected synchronized boolean parse(final Path file) throws XMLDBException {
```

Comment:
nope, I think that you are not right here. Will your code produce same result for `windows`, `linux` and `osx` with `windows` style path?